Zend/zend_language_parser.y

```diff
@@ -1241,7 +1246,7 @@ returns_ref:
 ;
 
 lexical_vars:
-		%empty { $$ = NULL; }
+		%prec PREC_ARROW_FUNCTION %empty { $$ = NULL; }
```

Comment:
This fixes a shift/reduce conflict, but I'm not sure about this. Need to dig more.